### ISSUE TITLE
ImportError: No module named pycosat

### ISSUE THREAD

**Issue by bphuyal** [conda/conda#3534]
ImportError: No module named pycosat

**Comment by gour** [comment]
Same here with minimal install:

```
$ conda list
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Traceback (most recent call last):
  File "/home/gour/.local/lib/python3.5/site-packages/conda/exceptions.py", line 472, in conda_exception_handler
    return_value = func(*args, **kwargs)
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/main.py", line 106, in _main
    p, sub_parsers = generate_parser()
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/main.py", line 55, in generate_parser
    from ..cli import conda_argparse
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/conda_argparse.py", line 15, in <module>
    from .common import add_parser_help
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/common.py", line 16, in <module>
    from ..resolve import MatchSpec
  File "/home/gour/.local/lib/python3.5/site-packages/conda/resolve.py", line 14, in <module>
    from .logic import minimal_unsatisfiable_subset, Clauses
  File "/home/gour/.local/lib/python3.5/site-packages/conda/logic.py", line 31, in <module>
    import pycosat
ImportError: No module named 'pycosat'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/gour/.local/bin/conda", line 9, in <module>
    load_entry_point('conda==4.2.7', 'console_scripts', 'conda')()
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/main.py", line 150, in main
    return conda_exception_handler(_main)
  File "/home/gour/.local/lib/python3.5/site-packages/conda/exceptions.py", line 488, in conda_exception_handler
    print_unexpected_error_message(e)
  File "/home/gour/.local/lib/python3.5/site-packages/conda/exceptions.py", line 436, in print_unexpected_error_message
    info_stdout, info_stderr = get_info()
  File "/home/gour/.local/lib/python3.5/site-packages/conda/exceptions.py", line 395, in get_info
    from conda.cli import conda_argparse
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/conda_argparse.py", line 15, in <module>
    from .common import add_parser_help
  File "/home/gour/.local/lib/python3.5/site-packages/conda/cli/common.py", line 16, in <module>
    from ..resolve import MatchSpec
  File "/home/gour/.local/lib/python3.5/site-packages/conda/resolve.py", line 14, in <module>
    from .logic import minimal_unsatisfiable_subset, Clauses
  File "/home/gour/.local/lib/python3.5/site-packages/conda/logic.py", line 31, in <module>
    import pycosat
ImportError: No module named 'pycosat'
```

**Comment by gour** [comment]
My mistake due to clash with pip.

**Comment by kalefranz** [comment]
This appears to be an example of conda "uninstalling" itself after a failed update.  The root cause was fixed in conda 4.3.
